sites/bioopticsworld.com/server/components/document.marko

```diff
@@ -11,11 +14,16 @@ $ const { site, req } = out.global;
     <marko-web-gam-init />
     <marko-web-gtm-init container-id=site.get("gtm.containerId") />
     <marko-web-native-x-init uri=nativeX.getUri() enabled=nativeX.isEnabled() />
+    <common-gam-prestitial init=true adunit=adunit />
     <${input.head} />
     <marko-web-gtm-start />
     <marko-web-gam-enable />
     <marko-web-gam-targeting key-values={ uri: req.path } />
   </@head>
+  <@body-wrapper enabled=true attrs={ id: "body-wrapper", class: "full-flex-column", style: wrapperStyle } />
```

Comment:
Full flex column wasn’t brought over in the CSS. I don’t believe this is included in core and was added specifically for informa.